### PR TITLE
update action

### PR DESCRIPTION
**Pull request by xiangyan99**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: .github/workflows/label-needs-triage.yml [hunk 1]
```diff
@@ -12,5 +12,4 @@ jobs:
       - name: add labels
         uses: actions/labeler@v3
         with:
-          github_token: ${{ secrets.GH_AZURE_SDK_PROJECT_PAT }}
```
Comment (scbedd) [inline .github/workflows/label-needs-triage.yml:15]: @xiangyan99 you can develop this in your fork. I highly recommend going that direction.

1. Open `actions` in your fork. Enable them if necessary.
2. Update your `main` or default branch. 
3. Create issue.

Everything should work, and you will be able to do mul;tiple attempts without roundtripping in upstream main.

Comment (xiangyan99) [reply .github/workflows/label-needs-triage.yml:15]: hah, yes, this was what I wanted to say. I could not open issues in my fork.

What I can do is merge the PR in my fork and submit some changes.

**Review by scbedd — APPROVED** [review]
I'll approve to unblock the test, but we gotta have a better way to test these. Maybe new private repo where you can rapidfire these attempts and freely create issues? At the end of the day you're just trying to process an event payload, and you don't even really care what that payload IS!

**Comment by xiangyan99** [comment]
> I'll approve to unblock the test, but we gotta have a better way to test these. Maybe new private repo where you can rapidfire these attempts and freely create issues?

Agreed we need some other ways to test actions.

**Comment by scbedd** [comment]
@xiangyan99 I do think repo on your account is the way to go here. Will be able to copy/pasta the action over here after you're done.

**Pull request merged**